lib/loggability/spechelpers.rb

```diff
@@ -7,7 +7,7 @@
 # Some helper functions for testing. Usage:
 #
 #    # in spec_helpers.rb
-#    RSpec.configure do |c|
+#    igure do |c|
```

Comment:
oops, not sure if we should just delete this whole comment, but some accidentally deleted text

Reply:
Ooops